src/search/sd-record-list.js

```diff
@@ -88,9 +88,13 @@ class SdRecordList extends GestureEventListeners(PolymerElement) {
     }
 
     goToDeeplink(e) {
-        console.log(e);
-        if (e.model.__data.item.record.deeplink !== "") {
-            window.open(window.location.origin + e.model.__data.item.record.deeplink, "_blank")
+        let deeplink = e.model.__data.item.record.deeplink;
+        if (deeplink) {
+            if (deeplink.startsWith("http")) {
+                window.open(deeplink, "_blank")
+            } else {
+                window.open(window.location.origin + deeplink, "_blank")
+            }
```

Comment:
`deeplink` doesn't change. No need to allow it to do so.
We can go without nested branching here and with a single action statement.
What about:
```
const deeplink = e.model.__data.item.record.deeplink;
if (!deeplink) return;

const targetLink = deeplink.startsWith("http") ? deeplink : window.location.origin + deeplink;
window.open(targetLink, "_blank");
```